libafl/src/mutators/token_mutations.rs

```diff
@@ -323,9 +323,12 @@ where
         let token_idx = state.rand_mut().below(tokens_len);
 
         let size = input.bytes().len();
+        // # Safety
+        // it's capped by max_size. can't overflow into 0
```

Comment:
What's the maximum max_size we allow? I think a user could definitely set this to max_usize?

Reply:
yeah it's a problem.
does it make sense to have saturating_add(1) here?

Reply:
instead of unwrap() or unwrap_unchecked

Reply:
Good point, `unsafe { NonZero::new(size.saturating_add(1)).unwrap_unchecked() }` is probably best